viewer/src/glsl/base/updateFragmentColor.glsl

```diff
@@ -53,7 +54,7 @@ void updateFragmentColor(int renderMode, vec4 color, float treeIndex, vec3 norma
     } else if (renderMode == RenderTypeDepth) {
         gl_FragColor = packDepthToRGBA(depth);
     } else {
-        gl_FragColor = vec4(1.0, 0.0, 0.0, 1.0);
+        gl_FragColor = vec4(1.0, 0.0, 1.0, 1.0);
```

Comment:
Is this supposed to be an error color? When does this even ever happen, and why purple?

Reply:
Just a safe guard should we introduce a new RenderMode and forget to update check (or provide wrong value somehow).